src/web/src/components/Schedule.vue

```diff
@@ -10,7 +10,7 @@
         v-for="(hour, index) of hours"
         :key="hour"
         :style="{ height: hourHeight + '%' }"
-      >
+      >    
```

Comment:
Yes definitely. I'm not even sure how that came about. Probably because I had the Image of just the Schedule but I moved the Code to include the CRN. Nevertheless, will be fixed.